plugins/DuskVerb/tools/tuner/ab_fdn_vs_accuratehall.py

```
@@ -0,0 +1,98 @@
+    "Bright Hall":          (f"{ANCH}/vvv-bright-hall","vvv-bright-hall"),
+}
+
+
+def full_rms(p):
+    x, _ = sf.read(p); m = x.mean(axis=1) if x.ndim > 1 else x
+    return float(np.sqrt(np.mean(m**2)))
+
+
+def render(name, algo10, dst):
+    for f in glob.glob(f"{OUTD}/*.wav"):
+        try: os.remove(f)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)